docs/TroubleShooting.md

```diff
@@ -54,6 +54,22 @@ make
 sudo make install
 ```
 
+### Issues with manual toolchain setup
+When ```make ``` executes, you may encounter below error:
+```
+arm-none-eabi-gcc: Command not found
+```
+This may happen if the toolchain path is not added to ```PATH```. To resolve:
```

Comment:
I mean, it is better to move `To resolve:` into the new line.
For this, you can add two spaces after ```PATH``` :)

Reply:
Done.